package.json

```
@@ -296,7 +296,7 @@
     "@elastic/eslint-config-kibana": "0.15.0",
     "@elastic/eslint-plugin-eui": "0.0.2",
     "@elastic/github-checks-reporter": "0.0.20b3",
-    "@elastic/makelogs": "^5.0.1",
+    "@elastic/makelogs": "^6.0",
```

Please, just for consistency sake 

```suggestion
    "@elastic/makelogs": "^6.0.0",
```